_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -207,7 +207,7 @@ The `aria-label` property is [prohibited][] for an element with a `generic` role
 This `div` element has no [WAI-ARIA state or property][].
 
 ```html
-<div role="region">A region of content</div>
+<div role="region" aria-label="Region name">A region of content</div>
```

Comment:
This makes the example Applicable (by having an `aria-*` attribute).

We should instead change the role into one that doesn't need any ARIA attribute.

Reply:
replaced with role="group", which doesn't require any aria state/property